homeassistant/components/matter/models.py

```diff
@@ -18,6 +18,14 @@
 ]
 
 
+# A sentinel object to detect if a parameter is supplied or not.
+class _UNSET_TYPE:
+    pass
+
+
+UNSET = _UNSET_TYPE()
```

Comment:
this is fine, you can also use `object()`

Reply:
Yeah, I liked the more explicit notation as hint in the discovery schemas when reviewing it by others